data/map planets.txt

```diff
@@ -838,6 +839,7 @@ planet Darkrest
 	attributes hai station uninhabited
 	landscape land/station34
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
+	description `	The original purpose of this station was to house Hai who had migrated from beyond the wormhole once it became clear that the Hai's empire was no longer tenable. However, today it is most notorious for an engineering malfunction that occurred when this station was still inhabited. In a freak accident, one of the station's hatches suddenly released oxygen out from the airlock, causing the deaths of countless civilians. Thousands of years later, no one knows for sure what caused the incident. Legends, stories, and conspiracy theories have blossomed from the event.`
```

Comment:
Just one hatch killed countless civilians? Maybe better to give a little bit more clarification; unless there was something blocking the Hai evacuating, it seems unlikely that a lot of people died because an oxygen leak went unnoticed. What if an entire section lost power at once and simultaneously depressurised? This might prevent evacuation and could also be more mysterious than a simple oxygen leak.

(I don't have much knowledge about depressurisation so if someone could corroborate me on what I've said, that'd be great. 👍) 
```suggestion
	description `	The original purpose of this station was to house Hai who had migrated from beyond the wormhole once it became clear that the Hai's empire was no longer tenable. However, today it is most notorious for an engineering malfunction that occurred when this station was still inhabited. In a freak accident, an entire habitation section of Darkrest lost power and depressurized, causing the deaths of countless civilians. Thousands of years later, no one knows for sure what caused the incident. Legends, stories, and conspiracy theories have blossomed from the event.`
```

Reply:
I found it plausible enough that one leak cascaded via explosive decompression under just the right (wrong?) conditions, but this does make that outcome a lot more reasonable.